appmgr/src/nginx/main-ui.conf.template

```diff
@@ -19,7 +21,8 @@ server {{
         gzip on;
         gzip_vary on;
         gzip_min_length 1024;
-        gzip_types text/plain text/css text/xml text/javascript application/x-javascript application/xml;
+        gzip_types text/plain text/css text/xml text/javascript application/javascript image/svg+xml font/tts font/otf font/eot font/openttype application/x-javascript application/xml;
```

Comment:
![image](https://user-images.githubusercontent.com/2364004/148109808-141af030-a842-4f10-be11-03b2b75b6aee.png)
The js and the xml where not compressed